plugins/optimization-detective/detect.js

```diff
@@ -424,8 +471,6 @@ export default async function detect( {
 		} );
 	}
 
-	const { onLCP } = await import( webVitalsLibrarySrc );
-
 	/** @type {LCPMetric[]} */
 	const lcpMetricCandidates = [];
 
```

Comment:
Now that `onLCP` is typed as `OnLCPFunction` I shouldn't think that `/** @type LCPMetric */` is needed here:

https://github.com/WordPress/performance/blob/47d06dc669b34ef2cffaef01ad8be2d11cbe06ca/plugins/optimization-detective/detect.js#L479-L483

However, when I remove it, TypeScript in PHPStorm complains:

![image](https://github.com/user-attachments/assets/79c7bd92-6506-40c0-a1cc-9aa30e7d91c1)

It doesn't complain about the usage in Image Prioritizer's `detect.js` here, however:

![image](https://github.com/user-attachments/assets/14acd817-3efa-4e20-997b-803375af34e6)